### DISCUSSION TITLE
Creating a singleExe from UnPackaged app breaks in latest WindowsApp SDK

### DISCUSSION THREAD

**Discussion by sudharsan-narayanan** [microsoft/WindowsAppSDK#4389] [Q&A]
Currently using the latest SDK 1.5.240428000, it fails to generate the `resource.pri` when creating the SingleExe. I think, it is breaking the latest SDK behavior. 

If we use the SDK version 1.4.240211001, It creates the SingleExe properly with all dependency file.

It's replicate in the CustomControl library project also. Find the sample and screenshot below,

[CustomControl.zip](https://github.com/microsoft/WindowsAppSDK/files/15219997/CustomControl.zip)
[SimpleApp.zip](https://github.com/microsoft/WindowsAppSDK/files/15220007/SimpleApp.zip)
 

In Version `1.4.240211001`

![image](https://github.com/microsoft/WindowsAppSDK/assets/93642597/81852428-1042-49d9-9f66-63ecd5470e59)

In Version `1.5.240428000`

![image](https://github.com/microsoft/WindowsAppSDK/assets/93642597/b335ae41-7235-4636-8f91-5f39d67e7caa)

**Comment by codendone** [comment]
Sounds like a bug, which is being tracked by #4390.
